### PR TITLE
Remove unused functions from RobotModel

### PR DESCRIPTION
**Pull request by abihall**
**Removes unused RobotModel Functions**

This PR removes the unused functions found in RobotModel.cpp which include `getCylindricalPostCamSpaceNormal` and `getSquarePostCamSpaceNormal`. These currently return the Zero vector due to `Goal::MeasurementType` enumeration and are never used. 

This is related to issue #705

### CODE REVIEW

**Comment by KipHamiltons** [comment]
I am very keen to remove these. They're properly dead. They are not called and even if they were, they don't apply to the only measurement type we actually use. On top of that, no one can on the team can confidently say they understand the internals because I spent months trying to get an insight when I was Eigenifying it because the logic was _invalid_ during its arma iteration (it would throw an exception if it was ever run, which it wasn't). I ended up piecing together what I thought the intentions of these functions was (after sitting down with @tayloryoung6396) and trying to approximate that with Eigen, but that means that we should have absolutely zero confidence about their correctness. If they had tests, we could have some, but they don't.

Basically, it's unused and almost certainly buggy code that no one fully understands. I won't approve without either @TrentHouliston or @Bidski 's blessing, but IMO this is a clear cut case.

**Review by TrentHouliston — COMMENTED** [review]
I don't mind these being removed.
For what they are supposed to do, they are supposed to work out what the normal vector should be from your camera of each side of the goal post, for both cylindrical goals and square goals. THey went along with the other measurement types when we tried to use the normals as measurements (didn't work so well)

**Review by KipHamiltons — APPROVED** [review]
LGTM!

**Pull request merged**
